src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,5 +1,20 @@
 package mate.academy
 
+import mate.academy.service.FigureSupplier
+
+const val ARRAY_SIZE = 6
+
 fun main() {
-    // draw all figures here
+    val array = Array(ARRAY_SIZE) { FigureSupplier().getDefaultFigure() }
+    for (i in 0 until array.size - 1) {
+        if (i < array.size / 2) {
+            array[i] = FigureSupplier().getRandomFigure()
+        } else {
+            array[i] = FigureSupplier().getDefaultFigure()
+        }
+    }
+
+    for ((index, figure) in array.withIndex()) {
```

Comment:
Please consider using of forEachIndexed
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/for-each-indexed.html